site/en/guide/function.ipynb

```diff
@@ -1113,7 +1112,7 @@
       "source": [
         "## Further reading\n",
         "\n",
-        "To learn about how to export and load a `Function`, see the [SavedModel guide](../../guide/saved_model). To learn more about graph optimizations that are performed after tracing, see the [Grappler guide](../../guide/graph_optimization). To learn how to optimize your data pipeline and profile your model, see the [Profiler guide](../../guide/profiler.md)."
+        "To learn about how to export and load a `Function`, refer to the [SavedModel guide](../../guide/saved_model). To learn more about graph optimizations that are performed after tracing, check the [Grappler guide](../../guide/graph_optimization). And, to learn how to optimize your data pipeline and profile your model, the [Profiler guide](../../guide/profiler.md) can help."
```

Comment:
To learn how to optimize your data pipeline .... (keep original)

Reply:
Gotcha. I'd still prefer to replace"see" with "check" or "go to" for accessibility (inspired by @heyawhite), if that's OK with you.